### ISSUE TITLE
fxLayout.xs="column" issue on iPhone

### ISSUE THREAD

**Issue by bfwg** [angular/flex-layout#295]
```
<div fxLayout="row" fxLayout.xs="column" fxLayoutAlign="center none">
  <app-api-card fxFlex></app-api-card>
  <app-api-card fxFlex></app-api-card>
  <app-api-card fxFlex></app-api-card>
</div>
```
The expected result for above code should be:

<img src="https://cloud.githubusercontent.com/assets/12819525/26331701/35f1f270-3f07-11e7-9b22-521cc57de8b8.png" alt="Spring Boot and Angular 2" width="250" height="450"/>

But instead, I got this on my iPhone. When `fxLayout.xs="column"` is removed, everything works fine.
<img src="https://cloud.githubusercontent.com/assets/12819525/26331888/465668c0-3f08-11e7-929e-1cfbd2853e3c.png" alt="Spring Boot and Angular 2" width="250" height="450"/>

Is this a bug? Any help would be much appreciated.

**Comment by jeffbcross** [comment]
Known issue: https://github.com/angular/flex-layout/issues/201

TL;DR: In Safari, if parent elements don't have a set height, this kinda thing happens.

**Comment by ThomasBurleson** [comment]
https://github.com/angular/flex-layout/issues/201#issuecomment-310517875

**Comment by ThomasBurleson** [comment]
Closing as "not a flex-layout issue"; this is a known flexbox column layout issue/workaround.
